src/v2/components/Validators/Table/index.jsx

```diff
@@ -20,47 +20,45 @@ import NodesStore from 'v2/stores/nodes';
 
 import useStyles from './styles';
 
-const ValidatorsTable = ({vertical}: {vertical: boolean}) => {
+const ValidatorsTable = ({separate}: {separate: boolean}) => {
   const classes = useStyles();
   const theme = useTheme();
   const showTable = useMediaQuery(theme.breakpoints.up('md'));
   const {
-    cluster: {nodes},
+    cluster: {voting = []},
   } = NodesStore;
   const renderRow = row => {
     return (
-      <TableRow hover key={row.pubkey}>
+      <TableRow hover key={row.nodePubkey}>
         <TableCell align="center">
-          <div className={classes.name}>
+          <Link to={`validators/${row.nodePubkey}`} className={classes.name}>
             <span />
-            {row.pubkey}
-          </div>
+            <div>{row.nodePubkey}</div>
+          </Link>
         </TableCell>
-        <TableCell>total sol</TableCell>
-        <TableCell>Commission</TableCell>
-        <TableCell>Uptime</TableCell>
+        <TableCell>{row.stake} Lamports</TableCell>
+        <TableCell>{row.commission}</TableCell>
       </TableRow>
     );
   };
   const renderCard = card => {
     return (
-      <div className={cn(classes.card, vertical && classes.cardVertical)}>
-        <div className={classes.name}>
+      <div
+        className={cn(classes.card, separate && classes.cardVertical)}
+        key={card.nodePubkey}
+      >
+        <Link to={`validators/${card.nodePubkey}`} className={classes.name}>
```

Comment:
I think the links should be absolute if possible because there is cross-linking from detail to table to overview etc.

Reply:
fixed